src/App.tsx

```diff
@@ -1,42 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
-export const tabs = [
+import { Tab } from './interface/interface';
+
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const tabSelect = (tab: Tab) => {
+    setSelectedTabId(tab.id);
+  };
```

Comment:
You don't need to pass the whole tab here. Just pass the id
```suggestion
  const tabSelect = (id: string) => {
    setSelectedTabId(id);
  };
```

Reply:
![image](https://user-images.githubusercontent.com/98056446/230746616-40912760-8f0b-45ba-97b7-9cd39616f610.png)
If I don't pass the whole tab, one of the test will fail